CMakeLists.txt

```diff
@@ -352,9 +377,29 @@ endif()
 
 
 if(WIN32)
-	file(GLOB dep_files
-		${dep_files}
-		"${CMAKE_FIND_ROOT_PATH}/bin/*.dll")
+	#Conan imports enabled
+	if(DEFINED CONAN_CXX_FLAGS)
```

Comment:
Conan is already here

Reply:
There is no Conan here, so, I can remove this flag.

Reply:
fixed